src/sources/kafka.rs

```diff
@@ -272,14 +296,14 @@ mod integration_test {
         test_util::{collect_n, random_string},
         Pipeline,
     };
-    use chrono::Utc;
+    use chrono::{SubsecRound, Utc};
     use rdkafka::{
         config::ClientConfig,
         producer::{FutureProducer, FutureRecord},
         util::Timeout,
     };
 
-    const BOOTSTRAP_SERVER: &str = "localhost:9092";
+    const BOOTSTRAP_SERVER: &str = "localhost:9091";
```

Comment:
Just curious, why this change?

Reply:
The makefile is starting kafka with a platintext listener on 9091, 9092 is SSL.

```
 587   │      -e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9091,SSL://localhost:9092,SASL_PLAINTEXT://localhost:9093 \
```

I suppose we could also look to setup vector with a cert and use 9092 :thinking:

Reply:
I personally think PLAINTEXT is fine for this test but we could add other tests later for SSL.

Reply:
Probably worth another issue/pr to test the TLS config for the source

Reply:
Ah, that makes sense. Yeah, I'm fine to use plaintext for most tests and just have one that exercises TLS.